### ISSUE TITLE
Backend language?

### ISSUE THREAD

**Issue by CatsAreFluffy** [inkron/Server#1]
Python is a good idea IMO.

**Comment by encloinc** [comment]
Pythons great, but I'm not sure if it's better than node. I definitely have more experience with python though

**Comment by CatsAreFluffy** [comment]
At any rate, we should probably get more people on the project before committing to a language.

**Comment by encloinc** [comment]
I agree, then we can balance the languages based on whats more popular amongst our developers

**Comment by pen-and-paper** [comment]
Are we going to allow the use of BBCode in the Inkron chat? That'd be pretty cool...

**Comment by CatsAreFluffy** [comment]
BBCode is a bit bulky for a chat platform. Markdown would probably work better.

**Comment by encloinc** [comment]
eh, is markdown like the _hello_

**Comment by CatsAreFluffy** [comment]
Yep.
